tests/test_cli.py

```diff
@@ -32,13 +48,48 @@ def test_help(self):
         self.assertEqual(cproc.returncode, 0)
 
     def test_show_settings(self):
-        env = os.environ.copy()
-        env['PYTHONPATH'] = os.pathsep.join(sys.path)
         cproc = subprocess.run([os.path.join(ROOT, 'zeek-client'), 'show-settings'],
                                check=True, env=self.env, capture_output=True)
         self.assertEqual(cproc.returncode, 0)
 
 
+class TestBundledCliInvocation(unittest.TestCase):
+
+    # Verify that zeek-client finds its package in Zeek-bundled install, where
+    # the package will not be in the usual Python search path. As we add more
+    # system-level testing, this may become a btest too, but for we stick to
+    # Python. Most system-level testing happens in the zeek-testing-cluster
+    # external testsuite.
+    @unittest.skipUnless(shutil.which('cmake') and shutil.which('make'),
+                         'needs both cmake and make in the system path')
+    def test_bundled_install(self):
+        with tempfile.TemporaryDirectory() as tmpdir, setdir(tmpdir):
+            # Configure the package via cmake with a Python module directory, as
+            # Zeek would do. Do this from the temp directory we're now in ...
+            cproc = subprocess.run(
+                ['cmake',
+                 '-D', f'PY_MOD_INSTALL_DIR={os.path.join(tmpdir, "python")}',
+                 f'--install-prefix={tmpdir}',
+                 f'{ROOT}'],
+                check=True, capture_output=True)
+            self.assertEqual(cproc.returncode, 0, 'cmake invocation failed')
```

Comment:
This assertEqual() and the one below won't trigger due to passing `check=True` part of the `run()` invocation, cp.returncode is always `0` if we get here.

Suggest to remove the assertEqual()s as it's only part of the test setup.